util/auth.py

```diff
@@ -208,5 +207,9 @@ def send_post_consent(self):
 
             html_previewer.preview(html)
 
+    def get_account_by_email(self, email):
+        # Retrieve an account by email address using the postgres_db module.
+        return get_account_by_email(email)
+
```

Comment:
@kluver Does this way of fetching accounts by email address make sense to you? I'm a little puzzled because I don't see that putting a method here does anything to restrict access and I'm not sure if we need a separate utility for fetching accounts by email address or if an existing repository method would work here. That might just be because I'm less familiar with the auth code though